docs/daily-usage.md

```diff
@@ -424,6 +424,29 @@ In order to debug JavaScript code, you need to:
    and click "Run". The program will stop in any breakpoint you have and you will be
    able to analyze your JS environment.
 
+If you face some problems while debugging in Firefox or Chrome, see our [FAQ](faq.md) or
+the corresponding extensions documentation before opening an issue here.
+
+### Debugging Python + JS code 🤯
+
+In order to debug Python and JavaScript code at the same time, you need to:
+
+1. Install suggested extensions, as mentioned above
+2. Run the project with the provided `invoke` commands. (This will automatically
+   generate the VSCode configurations for you) **Note**: As the configurations are
+   defined per project and depending on the Odoo addons you are working on, you might
+   need to update your configuration frequently with `invoke write-code-workspace-file`
+   (done automatically when you do `invoke start`, for example).
```

Comment:
The launchers that start with "Start Odoo" already do this for you, so you can remove this step:

```suggestion
```